template.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 # Hack for https://github.com/rails/rails/issues/35153
 gsub_file 'Gemfile', /^gem ["']sqlite3["']$/, 'gem "sqlite3", "~> 1.3.0"'
-gem 'hyrax', '3.0.0-rc2'
+gem 'hyrax', '3.0.0.pre.rc2'
```

Comment:
this shouldn't need to change. bundler/rubygems supports both formats interchangeably: https://github.com/samvera/hyrax/pull/4467#issuecomment-663642080